packages/api/db/migration/20231024164318_null_agricuture_associated.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
So, lol .. my interpretation of what Kevin was asking was -- "migrations are a pain/should be used sparingly so if we can remove this without affecting the db or schema that would be sweet" 

Just like the commented code and if we are doing a migration anyways... should we just remove it and add it back in with the rest of the "ready-to-go" PR since it is just as easy to dropColumn? We can set preexisting custom types to null at that point too if that has some usefulness. 

There are so many unused ghost tables and columns in the db that should be re-evaluated too.  For example in task table there is a "photo, and action_needed" alluding to unimplemented features of the past.

Reply:
Hmmm, I'm not sure about that. If you read the full Slack thread, Kevin mentioned several times that he wanted the column to stay in the database specifically with value `null` value. I would prefer to write the migration that Kevin asked for, since it was the migration that Kevin asked for!

I was also thinking of the weird orphan unused columns that sprinkle the database (ugh), but the thread says this is a funded module and that question is relevant to research, so I don't think it's going to be abandoned entirely.

Reply:
I am good with @antsgar  deciding best practice here and me just approving. But I am seeing nothing about the importance of keeping the column for this rollback in that Slack thread so much as generally when making db columns providing the option for null vs default(true) -- and setting null when the value is unknown. 

Kevin:

> In the general case, I think it's really valuable to have null, false, true for data points. Otherwise you can make assumptions around the shape of the data that are actually just default values. .......  there's a lot of hands in the pot, so while we may "know" that this true doesn't mean anything, someone forking LiteFarm for their own uses may not.